site/src/component/AppHeader/Profile.tsx

```diff
@@ -27,15 +27,15 @@ const Profile = () => {
   const DefaultTab = (
     <>
       <div className="profile-popover__header">
-        <img src={picture} alt={name} />
-        <span>
-          <h1 title={name}>{name}</h1>
```

Comment:
Do you have an example of a name/email that would do this? The font size was changed so more text should fit in the box, and I also tested it with a few longer names. I will put this back though since that reason makes a lot of sense to me.

Reply:
Any full name from a culture that uses 4 names would probably overflow here easily.